charts/prometheus/values.yaml

```
@@ -65,7 +65,8 @@ configmapReload:
       digest: ""
       pullPolicy: IfNotPresent
 
-    # containerPort: 9533
+    containerPort: 8080
+    containerPortName: metrics 
```

A comment will help:
```suggestion
    ## config-reloader's container port and port name for probes and metrics
    containerPort: 8080
    containerPortName: metrics 
```